Gemfile

```diff
@@ -162,3 +146,30 @@ unless ENV["APPLIANCE"]
     gem "rspec-rails",      "~>3.5.0"
   end
 end
+
+#
+# Custom Gemfile modifications
+#
+
+# Load developer specific Gemfile
+#   Developers can create a file called Gemfile.dev.rb containing any gems for
+#   their local development.  This can be any gem under evaluation that other
+#   developers may not need or may not easily install, such as rails-dev-boost,
+#   any git based gem, and compiled gems like rbtrace or memprof.
+#
+#   To develop a gem locally and override its source to a checked out repo
+#   you can use this helper method in Gemfile.dev.rb e.g.
+#
+# override_gem 'manageiq-ui-classic', :path => File.expand_path("../manageiq-ui-classic", __dir__))
+#
+def override_gem(name, *args)
+  return unless (dependency = dependencies.find { |d| d.name == name })
```

Comment:
Don't like the `return nil` here...
I think it should `raise` when you try to override a gem that is not in the list. I'm assuming that this will silently move on.(maybe it was a typo)
Alternatively, it could just `gem name, *args` if not already listed.